plugins/easyimage/plugin.js

```diff
@@ -202,6 +202,9 @@
 	}
 
 	function registerUploadWidget( editor ) {
+		var objectUrlCache = {},
```

Comment:
I'm wondering if it won't be better to create nice class for managing object URLs.

Reply:
Yes, it will, but as I discussed with @mlewand we wanted the "minimalistic" approach here and creating proper manager (probably integrated somehow with `uploadwidget`) will be a follow up task.

Reply:
Proper manager will be implemented together with Mateusz hackathon task. Which will be as @f1ames mentioned a followup task, as it will require much more time than the simple one.